src/applications/edu-benefits/pages/contactInformation.js

```diff
@@ -35,7 +35,7 @@ export default function createContactInformationPage(
       'view:otherContactInfo': {
         'ui:title': 'Other contact information',
         'ui:description':
-          'Please enter as much contact information as possible so we can get in touch with you, if necessary.',
+          'Please enter your contact details below so we can get in touch with you, if necessary.',
```

Comment:
this will affect multiple forms

Reply:
Updated, revert those changes and updated the 10203 `personalInformation.js`